Calculator/calculator.py

```diff
@@ -0,0 +1,6 @@
+def sum(num1,num2):
+    return num1+num2
+
+#Testing sum
+seven = sum(1,6)
```

Comment:
Just curious:

What is the result of this call?

`sum("1", "6")`

This method allow numbers or strings? Can you validate please?

Reply:
This is done, I already add the changes

Reply:
Ok this is good but.. This method only allow `int` values? What about `float` values? And also what about negative values? What is the result of `sum(-1, 6)` is this ok?